### DISCUSSION TITLE
boring-marriage-tradition/

### DISCUSSION THREAD

**Discussion by giscus** [amir-pourmand/hugo-papermod-farsi-template#37] [Announcements]
# boring-marriage-tradition/

این مطلب صرفاً یک دل‌نوشته است برای گفتن حرف‌هایی که دوست داشتم گفته بشه.
بیاید یک شب عروسی خیلی عادی رو از نظر خودم براتون تعریف کنم. من همچین شبی رو بارها تجربه کردم و هر بار کمابیش یکسان بوده. چیزهایی که می‌نویسم، طنز تلخه. ولی از اون جنس‌‌هایی که راجع بهش صحبت نمیشه.
جالب این که بعد از نوشتن مطلب با دوستانم راجع به این نوع عروسی صحبت کردم و گفتند که آره.

https://aprd.ir/boring-marriage-tradition/

**Comment by hiwasham** [comment]
امیر جان نوشته جالبی بود از این نظر که تامل کردی روی چیزهایی که معمولا عادت داریم بهشون توجه نکنیم در حالیکه وقتی به اینها توجه میکنیم یعنی انگار به همه جوانب عادتها و هنجارها و رسومات داریم توجه میکنیم. نتیجه این توجه هم معمولاً پیدا کردن تناقضها و باگهاست.
من با این سیستم عروسی‌ای که گفتی زیاد آشنایی ندارم. فقط یک بار تهران شرکت کردم زیاد هم به این موارد توجه نکردم ولی تقریباً به همین شکلی بود که گفتی.
عروسی های ما (کردها) چند تا تفاوت داره شاید از این نظر کمی بیشتر خوش بگذره. اول اینکه زن ها و مردها با هم هستند. البته خیلی ها با این جمله یاد این میفتن که مردها و زن های غریبه توی هم هستن. در حالیکه مهمترین جنبه‌اش اینه که ما نزدیک مادر و خواهر و دختر عمو و ... خودمون هستیم و همینطور بقیه. غریبه و آشنا هم خیلی مطرح نیست. آدمها راحت با هم حرف میزنن یا به هم کار دارن بدون اینکه عجیب به نظر برسه. این غریبه و آشنا بودن رو بعداً در شهرهای دیگه دیدم. 
یه مورد دیگه هم رقص کردی هست که هم به شکل گروهیه هم به در اون زن و مرد جدا نمیشن. خانمها هم اجباری برای پوشیدن حجاب ندارن.
یه سری ریزه کاری ها و جزئیات هم داره که دیدم بقیه براشون جالبه. مثلاً یه مرد که به حلقه رقص ملحق میشه، باید بین دو مرد بره (یعنی دست زن رو نباید بگیره). اما دختر میتونه جایی بره که پسر هم هست. یا اگه دختری اومد توی رقص دستت رو گرفت نباید زودتر از اون بری. باید صبر کنی اول اون چند دور برقصه و تصمیم بگیره که از حلقه رقص بره بیرون بعدش پسر میتونه بره. البته اگه کسی اینهارو رعایت کنه اتفاق بدی نمیفته فقط ممکنه بقیه تعجب کنن یا این رو یه سیگنال منفی بدونن.
یادمه نوجوون که بودیم برای این موضوعات هیجان زیادی داشتیم و دنبال این بودیم که فرصتهایی رو فراهم کنیم که چند دور با دختری که دوستش داشتیم برقصیم. یه هماهنگی هایی انجام میدادیم یا سیاستهایی به خرج میدادیم که فرصتش فراهم بشه که فلان دختر بیاد تو دست ما (حتی برای دختری که در چنین فرصتی میاد کنارت هم کلمه وجود داره. یه چیزی شبیه کراش)
همین چند روز پیش هم که مراسم نوروز بود در ده ها روستا و نقاط مختلف شهرهای کرد نشین چنین مواردی وجود داشت. منم چند تاشو حضوری دیدم. هزاران نفر دختر و پسر جوون تا مردها و زناهای میانسال بدون پوشش اجباری و روسری و ... بدون اینکه مشکلی پیش بیاد حلقه های رقص تشکیل میدادن. از افطار تا 1-2 بامداد.

**Reply by pourmand1376** [reply]
سلام هیوا جان. ممنون که توجه کردی. این پست رو ۴ ۵ ماه پیش هنگام عروسی یکی از فامیل‌ها نوشتم و چون اون زمان نمی‌شد منتشر کنم، گذاشتم چند ماه بعد منتشر بشه. :)

سبک عروسی‌ای که توصیف کردی، به نظر عروسی جالبی میاد و دوست دارم قسمت بشه روزی از این نوع عروسی‌ها هم شرکت کنم. یک بار بخشی از فیلمش رو تو گوشی آشناها - که شرکت کرده بود - دیدم و می‌گفت کلی هم جالب بوده. اون عروسی داخل شیراز بود. 

راجع به دغدغه حجاب که مطرح کردی، راستش فکر نمی‌کنم دغدغه اصلی کسانی که عروسی مکه‌ای برگزار می‌کنند، حجاب باشه؛ چون معمولاً از این که همه بدون حجاب جلوی داماد برقصند مشکلی ندارند. انگار بحث محرم یا نامحرم جلوی داماد مطرح نیست. 

فکر می‌کنم این بیشتر یک رسمی هست که جا افتاده و هیچ کس هم نمی‌دونه چرا به این شکل مسخره جاافتاده و چون مردها اصولاً اعتراضی راجع به نحوه برگزاری این مراسم‌ها ندارند، همین شکل مونده. 

نوشتن راجع به این قضیه هم وقتی به ذهنم رسید که دیدم خانم‌ها معمولاً از مراسم عروسی خیلی تعریف می‌کنند و وقتی برمی‌گردند با کلی ذوق و شوق می‌شینند راجع به کارهایی که تو مراسم انجام شده، صحبت می‌کنند؛ در حالی که من به شخصه هیچ‌وقت چیز جالبی در مراسم عروسی ندیدم. مگر این که نشستم با یکی صحبت کردم که طرف آدم جالبی بود. 

اینجا بود که متوجه شدم که این مراسم اکثراً فقط برای خانم‌ها برگزار میشه و ما چندان داخل آدم حساب نمی‌شیم و همین هست که مراسم برای اون‌ها جذابه ولی برای ما نیست. 

پی‌نوشت: یادم اومد که ما هم تو برخی روستاهای یزد (از جمله روستای پدربزرگم - هامانه) یک مدل عزاداری حلقه‌ای رو داریم که افراد دور هم می‌چرخند و به روش خاصی عزاداری برگزار می‌کنند ولی تا به حال نشنیدم جایی تو یزد، به این شکل جشن بگیرند.

**Comment by amirghh** [comment]
سلام
نه لزوما ربطی به قومیت نداره. توی فامیل ما از قدیم اینطور بود که عروسی سمت مادری مختلط برگزار میشه و همه راحت هستند. حوصله ات سر نمیره و از اول تا آخر مراسم در حال رقص و شادی و صحبت کردن هستی. یعنی اونقدری زود میگذره که شاکی هستی چرا عروسی ها الان شده از ساعت 19-24. اما خوب طرف پدری کمی سنتی تر هستن و از قدیم مدل جدا از هم برگزار میشد که جدیدا بعضی از اونا هم دارن میرن سمت مختلط شدن.
شخصا توی عروسی فامیل هر چقدر نزدیک باشه حتی عمو و خاله و.. اگر با مراسم مختلط مشکل داشته باشه یا بخواد بیاد اما بشینه یه گوشه زیر چادر و در عذاب باشه ترجیحم اینه دعوت نکنم. خیلی هم رک و راست میگم جشن مختلط هست و بعنوان میزبان از همه مهمانها درخواست دارم توی جشن شاد باشن و بزنن و برقصن اگر احساس میکنید معذب خواهید بود تشریف نیارید. ترجیح میدم چه فایمیل چه غیر فامیل بیشتر رنج سنی خودمون رو دعوت کنم که شبیه یه پارتی خودمونی باشه که برات خاطره ساز باشه نه اینکه بعد 10 سال که فیلم عروسی رو می بینی فقط موضوع غذا خوردن و قایم شدن افراد از دوربین زیر چادر باشه :-))

**Comment by yjavaherian** [comment]
واقعا همینه. اکثر عروسی هایی که من هم دعوت شدم همینطور بوده و اصلا حال نمیده.
معمولا کاری که میکنم اینه که از سالون میزنم بیرون و اگه دوستی باشه با اون و اگر هم نه تنها، با هم راه میریم تو محوطه خارج سالون و حرف می زنیم.
